framework/src/modules/interoperability/constants.ts

```diff
@@ -43,6 +43,7 @@ export const MAX_MODULE_NAME_LENGTH = 32;
 export const MIN_CROSS_CHAIN_COMMAND_NAME_LENGTH = 1;
 export const MAX_CROSS_CHAIN_COMMAND_NAME_LENGTH = 32;
 export const CHAIN_ID_LENGTH = 4;
+export const MIN_RETURN_FEE_PER_BYTE_LSK = BigInt(1000);
```

Comment:
I think the name needs to be updated @Phanco can provide input here

Reply:
https://github.com/LiskHQ/lips/issues/281

Likely this issue will be accepted by research team. I think we can use the new suggested name in advance?